src/client.js

```diff
@@ -20,9 +20,10 @@ export class UiClient {
     }
 
     /**
+     * @param options `option.redirectUrl` the page which the confirmation email redirects to.
      *
-     * @param options `option.redirectUrl` is the URL to which the confirmation email redirects.
-     *                `option.returnToAfterEmailConfirmation` is where the page after the conformation email will redirect.
+     *                `option.returnToAfterEmailConfirmation` the confirmation email will redirect to one page.
+     *                This parameter is where this page will redirect.
```

Comment:
Not a fan of the latest version :/

Reply:
I know, this purpose of this parameter is hard to explain.